dango/types/src/bank/events.rs

```diff
@@ -0,0 +1,17 @@
+use {
+    grug::{Addr, Denom, Uint128},
+    std::collections::BTreeMap,
+};
+
+#[grug::derive(Serde)]
+#[grug::event("balance_changes")]
+pub struct BalanceChangesEvent {
```

Comment:
can you remove the word "Event" from the struct name? to be consistent with other events in the codebase.